app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -103,6 +105,7 @@ function InPersonLocationStep({ onChange, toPreviousStep }) {
       const selectedLocation = locationData[id];
       const { name: selectedLocationName } = selectedLocation;
       setSubmitEventMetadata({ selected_location: selectedLocationName });
+      setSubmitEventMetadata({ in_person_cta_variant: inPersonCtaVariantActive });
```

Comment:
can we combine the 2 lines so it's : setSubmitEventMetadata({selected_location: sla, in_person_cta_variant: ipcva})

Reply:
Done here too! Thank you :)